Modules/timemodule.c

```diff
@@ -1815,9 +1841,13 @@ PyInit_time(void)
 #endif
 
     if (PyErr_Occurred()) {
```

Comment:
Is this block still needed? I think we've checked every place that can raise, right?

Reply:
Hm, I think we have checked the error that we have know in this function, but I am not sure there have other errors we don't know or not:(

Reply:
We guard every single API call here; there's no way an error could have magically occurred (unless it wasn't detected by the APIs, but it's not our responsibility to protect against that). It should be safe to ditch this block.

Reply:
Make sense. Thanks, brandt.